app/controllers/review_mapping_controller.rb

```diff
@@ -272,39 +274,40 @@ def list_mappings
     # ACS Removed the if condition(and corressponding else) which differentiate assignments as team and individual assignments
     # to treat all assignments as team assignments
     @items = AssignmentTeam.where(parent_id: @assignment.id)
-    @items.sort {|a, b| a.name <=> b.name }
+    #@items.sort {|a, b| a.name <=> b.name }
```

Comment:
Please delete original code, instead of comment it out. Git will record your modification.